packages/twenty-server/src/engine/metadata-modules/agent/agent.service.ts

```diff
@@ -18,6 +19,7 @@ export class AgentService {
     @InjectRepository(RoleTargetsEntity, 'core')
     private readonly roleTargetsRepository: Repository<RoleTargetsEntity>,
     private readonly agentChatService: AgentChatService,
+    private readonly agentRoleService: AgentRoleService,
   ) {}
 
   async findManyAgents(workspaceId: string) {
```

Comment:
not a fan of these wrapping method. they don't add a lot on top of the existing ORM. we will likely end up adding options to findManyAgents methods which is the purpose of the ORM